backend/controllers/advertisementController.ts

```diff
@@ -50,4 +50,16 @@ export class AdvertisementController {
       res.status(500).json({ message: 'サーバー内部のエラーが発生しました' })
     }
   }
+
+  async deleteAdvertisement(req: express.Request, res: express.Response) {
+    try {
+      const advertisementData = req.body
+      await this.advertisementApplicationService.deleteAdvertisement(
+        advertisementData.advertisementId,
+      )
```

Comment:
```suggestion
      const { advertisementId } = req.body
      await this.advertisementApplicationService.deleteAdvertisement(
        advertisementId,
      )
```

https://qiita.com/kamijodev/items/ca007a264a3a0f1d1c8b#%E3%82%AA%E3%83%96%E3%82%B8%E3%82%A7%E3%82%AF%E3%83%88%E3%81%AE%E5%88%86%E5%89%B2%E4%BB%A3%E5%85%A5
onjectの分割代入は頻度高く使うんで覚えましょう！